src/Model/Serializer/ValueObject/Calendar/OpeningHoursNormalizer.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Model\Serializer\ValueObject\Calendar;
+
+use CultuurNet\UDB3\Model\ValueObject\Calendar\OpeningHours\OpeningHours;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+final class OpeningHoursNormalizer implements NormalizerInterface
```

Comment:
I wonder how this is handled inside the model classes, does it make sense to also use it there?

Reply:
I don't understand your remark?

Reply:
The existing import code uses `OpeningHourNormalizer` but it also needs to convert `OpeningHours`, so does it make sense to reuse the `OpeningHoursNormalizer` in the importer?